### PR TITLE
Fix mixpanel split range gap when run in real time

### PR DESCRIPTION
**Pull request by taikhuu**
Add overlaping range when querying with split range
Update unit tests

### CODE REVIEW

File: lib/embulk/input/mixpanel.rb [hunk 1]
```diff
@@ -162,10 +162,10 @@ def run
         @dates.each_slice(task[:slice_range]) do |dates|
           ignored_record_count = 0
           unless preview?
-            Embulk.logger.info "Fetching data from #{dates.first} to #{dates.last} ..."
+            Embulk.logger.info "Fetching data from #{@dates.first} to #{dates.last} ..."
```
Comment (sakama) [inline lib/embulk/input/mixpanel.rb]: Use both of `@dates` and `dates` in same block is confusing?

**Review by sakama — COMMENTED** [review]
Put 1 minor comment.
Everything else looks good to me.

File: lib/embulk/input/mixpanel.rb (hunk 1 above)
Comment (taikhuu) [reply lib/embulk/input/mixpanel.rb]: Yes, I will change it to a different name

**Comment by taikhuu** [comment]
@sakama  I changed the variable name

**Review by sakama — APPROVED** [review]
LGTM

**Pull request merged**